app/Providers/AppServiceProvider.php

```diff
@@ -23,6 +24,13 @@ public function boot()
      */
     public function register()
     {
-        //
+        $this->app->bind('Abraham\TwitterOAuth\TwitterOAuth', function () {
```

Comment:
Hm don't we already have one twitter package? Was it not able to do what this does?